plugins/base/src/test/kotlin/content/params/ContentForParamsTest.kt

```diff
@@ -805,7 +801,7 @@ class ContentForParamsTest : BaseAbstractTest() {
             | /**
             | * a normal comment
             | *
-            | * @return empty string when [Hash Map](java.util.HashMap.containsKey) doesn't contain value.
+            | * @return empty string when [Hash Map][java.util.HashMap.containsKey] doesn't contain value.
```

Comment:
Out of curiosity: did you change it because the k2 tests failed or was it an unrelated fix while passing by?

Reply:
The former. `[Hash Map](java.util.HashMap.containsKey)` is an incorrect link.

Reply:
See https://kotlinlang.org/docs/kotlin-doc.html#links-to-elements

Reply:
yeah, I understand, but it doesn't answer the question :) Does K1 support such incorrect links?

Reply:
> Does K1 support such incorrect links?

Yes, that is why the test was passed.
But IDE and K2 do not support them. 
![image](https://github.com/Kotlin/dokka/assets/9636937/15583566-5f72-4d0c-b1a1-07de3bd798f3)